examples-bkup/chat.rs

```diff
@@ -172,3 +207,64 @@ fn print_ourinfo(crust: &mut Crust) {
         serde_json::to_string(&ourinfo).unwrap()
     );
 }
+
+fn parse_cli_args() -> CliArgs {
+    let matches = App::new("Simple chat app built on Crust")
+        .about(
+            "This chat app connects two machines directly without intermediate servers and allows \
+             to exchange messages securely. All the messages are end to end encrypted.",
+        )
+        .arg(
+            Arg::with_name("listening_port")
+                .help(
+                    "Optional server Crust will be listening for incoming connections ON. If \
+                     unspecified, random port will be used.",
+                )
+                .short("p")
+                .value_name("PORT")
+                .takes_value(true)
+                .validator(|v| v.parse::<u16>().and(Ok(())).or(Err("Invalid port".into()))),
+        )
+        .arg(
+            Arg::with_name("our_ip")
+                .help(
+                    "Our IP address to use when constructing our connection info. If unspecified \
+                     , Crust will try to use bootstrap node to determine our IP.",
```

Comment:
Probably `Crust` can be replaced with just `peer`
```suggestion
                     , peer will try to use bootstrap node to determine our IP.",
```